etc/kayobe/ansible/fix-hostname.yml

```diff
@@ -27,4 +27,5 @@
   vars:
     reboot_hosts: fix-hostname
     reboot_with_bootstrap_user: true
+    confirm_reboot: "yes"
```

Comment:
Open decision on if we should set this here. If we don't ci-multinode will need to be updated as it symlinks this as a hook: https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/2024.1/etc/kayobe/environments/ci-multinode/hooks/overcloud-host-configure/pre.d/10-fix-hostname.yml
Otherwise, auto-deployments of ci-multinode won't work.

Reply:
I'd be tempted to set confirm_reboot as an extra var in all CI environments